hack/local-up-cluster.sh

```diff
@@ -62,6 +62,9 @@ CLOUD_PROVIDER=${CLOUD_PROVIDER:-""}
 CLOUD_CONFIG=${CLOUD_CONFIG:-""}
 FEATURE_GATES=${FEATURE_GATES:-"AllAlpha=true"}
 
+# enable swagger ui
+ENABLE_SWAGGER_UI=${ENABLE_SWAGGER_UI:-false}
+
```

Comment:
May I ask the reason/use case you want to enable `ENABLE_SWAGGER_UI`? Personally I don't think it's a frequently used flag for developers (the main users of this scripts)